### PR TITLE
Chore/update nx (Angular 13.x)

### PR DESCRIPTION
**Pull request by frontend-specialisten**
## References

- #0000

### CODE REVIEW

File: migrations.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
```
Comment (dzhavat) [inline migrations.json:1]: question: is this file necessary?

**Comment by dzhavat** [comment]
https://github.com/Energinet-DataHub/greenforce-frontend/blob/669b0beb1a4956ab6e41e7e65cbc76cb467a5b7a/libs/dh/metering-point/feature-overview/src/lib/dh-metering-point-overview.component.spec.ts#L93-L103

@LayZeeDK This test [fails](https://github.com/Energinet-DataHub/greenforce-frontend/runs/4470030405?check_suite_focus=true#step:10:2055) with `Class constructor ComponentStore cannot be invoked without 'new'` error pointing to `local-router-component-store.ts` file. Do you have an idea what might be causing it?

image

File: libs/ui-watt/src/lib/components/button/watt-button.stories.ts [hunk 2]
```diff
@@ -73,7 +73,9 @@ Button.args = {
 const iconButtonTemplate = (args: Partial<WattIconButtonComponent>) =>
   `<watt-icon-button icon="${args.icon}" [disabled]="${args.disabled}" label="Something meaningful"></watt-icon-button>`;
 
-const IconButton: Story<WattIconButtonComponent> = (args) => ({
+const IconButton: Story<WattIconButtonComponent | WattButtonComponent> = (
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/components/button/watt-button.stories.ts:76]: question: Was there a problem with `WattButtonComponent` not being specified?

**Comment by LayZeeDK** [comment]
Did we provide the local component store in the component tree? Otherwise, try the GlobalRouterStore, it has a root provider.

File: libs/ui-watt/src/lib/components/button/watt-button.stories.ts (hunk 2 above)
Comment (frontend-specialisten) [reply libs/ui-watt/src/lib/components/button/watt-button.stories.ts:76]: Compilation error, if not specified we can't disable the desired controls because they are not part of WattIconButtonComponent

**Comment by frontend-specialisten** [comment]
> Did we provide the local component store in the component tree? Otherwise, try the GlobalRouterStore, it has a root provider.

@LayZeeDK Yes the component store is provided directly inside of the component

**Comment by LayZeeDK** [comment]
I'll look into it tomorrow unless we have sprint planning.

**Comment by dzhavat** [comment]
@frontend-specialisten We're already behind 😀 https://blog.nrwl.io/redesigned-starter-page-storybook-6-4-eslint-8-and-more-in-nx-13-3-5507d7378360

**Comment by frontend-specialisten** [comment]
> @frontend-specialisten We're already behind 😀 https://blog.nrwl.io/redesigned-starter-page-storybook-6-4-eslint-8-and-more-in-nx-13-3-5507d7378360

@dzhavat I will run the migrations again 😅

**Review by dzhavat — APPROVED** [review]
